internal/storage/storage_handle.go

```diff
@@ -250,5 +253,12 @@ func (sh *storageClient) BucketHandle(bucketName string, billingProject string)
 		bucketName:    bucketName,
 		controlClient: sh.storageControlClient,
 	}
+
+	if sh.clientProtocol == "grpc" {
+		if err := storage.CheckDirectConnectivitySupported(ctx, bucketName, sh.storageClientOpts...); err != nil {
```

Comment:
Do we know, how much time does it take? - Can we update the description with this information?